sklearnex/__init__.py

```diff
@@ -32,6 +32,7 @@
     "basic_statistics",
     "cluster",
     "config_context",
+    "covariance",
```

Comment:
@olegkkruglov I believe your example in the description is incorrect. It should be ```IncrementalEmpiricalCovariance```. Importing ```EmpericalCovariance``` can only be done from the sklearnex.preview package. @samir-nasibli This would have been caught by https://github.com/intel/scikit-learn-intelex/blob/main/sklearnex/tests/test_patching.py#L289 but it only looks for matching estimators, which is not yet the case because EmpiricalCovariance is still in preview and IncrementalEmpericalCovariance is a sklearnex-unique estimator.

Reply:
> @olegkkruglov I believe your example in the description is incorrect. It should be `IncrementalEmpiricalCovariance`. Importing `EmpericalCovariance` can only be done from the sklearnex.preview package. @samir-nasibli This would have been caught by https://github.com/intel/scikit-learn-intelex/blob/main/sklearnex/tests/test_patching.py#L289 but it only looks for matching estimators, which is not yet the case because EmpiricalCovariance is still in preview and IncrementalEmpericalCovariance is a sklearnex-unique estimator.

@icfaust thank you! I was confused by the description. Let's cover this with a test

Reply:
i think the test is not really necessary here. after moving EmpiricalCovariance out of preview this will be covered

Reply:
Discussed with @icfaust 
We acan adapt this test 
https://github.com/icfaust/scikit-learn-intelex/blob/dev/centralize_finite_assert/sklearnex/tests/test_common.py#L101
if he were to load all_estimators and then cross-reference to sklearnex.__all__
But this suggestion the scope for another PR.

Please @olegkkruglov make sure to create a ticket for this.